### PR TITLE
M48 12x4 lego ortholinear keyboard

### PR DESCRIPTION
**Pull request by alinelena**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mlego/m48/info.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "mlego/m48",
+    "keyboard_folder": "mlego/m48",
+    "url": "https://gitlab.com/drFaustroll/m65",
+    "maintainer": "alin elena",
+    "width": 12,
+    "height": 4,
```
Comment (drashna) [inline keyboards/mlego/m48/info.json:7]:
```suggestion
```

File: keyboards/mlego/m48/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = yes
```
Comment (drashna) [inline keyboards/mlego/m48/keymaps/via/rules.mk:2]: This isn't needed, as via automatically enables it. 
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/mlego/m48/readme.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# M48
+
+![M48](https://i.imgur.com/QLuCAlW.jpg)
```
Comment (fauxpark) [inline keyboards/mlego/m48/readme.md:3]:
```suggestion
![M48](https://i.imgur.com/QLuCAlWh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mlego/m48/m48.h [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+*/
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_4x12(K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311) \
+    {                                                                                                                                                                                                                                                                                                                     \
+        {K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011}, {K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111}, {K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211}, { K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311 }          \
+    }
```
Comment (noroadsleft) [inline keyboards/mlego/m48/m48.h:25]:
```suggestion
#define LAYOUT_ortho_4x12( \
    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, \
    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, \
    K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, \
    K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311  \
) { \
    { K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011 }, \
    { K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111 }, \
    { K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211 }, \
    { K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311 }  \
}
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — COMMENTED** [review]
I just realized your PR targets the `develop` branch, and I don't see any reason that it needs to do so. Was that a deliberate decision?

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**